### PR TITLE
Update Project details

### PR DESCRIPTION
**Pull request by eshaanagarwal**
## Description
- [x] Added Previewer and Discord Bot Details
- [x] Updated images of Lite-hai, Internship portal and Telegram bot

## Fixes
Fixes parts of #45 

## Types of changes

- [ ] Bug/Hot fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring Old Code ( non-breaking change replacing already existing code )
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** guidelines in repository.
- [x] My pull request has a descriptive title and is attached to an existing issue.
- [x] I haven't repeated the code and it follows established code style of repository.
- [x] My branch is up-to-date with upstream branch.
- [x] All new and existing tests passed.

### CODE REVIEW

File: content/index/projectData.yml [hunk 1]
```diff
@@ -25,6 +26,7 @@
     gh_user: IIT-BHU-InstiApp
     repo: IIT-BHU-app
     url: https://play.google.com/store/apps/details?id=com.iitbhu.litehai
+    img: litehai.webp
     desc: Lite Hai! is a mobile app that acts as a guide for all the gymkhana happenings at IIT BHU.
```
Comment (king-11) [inline content/index/projectData.yml:30]: Can add more info here

File: content/index/projectData.yml [hunk 2]
```diff
@@ -45,6 +47,7 @@
   - name: Telegram Bot
     i18n: ...
     url: https://github.com/COPS-IITBHU/telegram-bots
+    img: telegrambot.webp
     desc: The telegram bots developed and used by the COPS Dev Group.
     tech:
       - Kotlin
```
Comment (king-11) [inline content/index/projectData.yml:53]: Typescript and Vercel

**Review by king-11 — CHANGES_REQUESTED** [review]

**Comment by king-11** [comment]
I will provide image for placement portal, for bots we can call some commands and then show it, lite hai can have better image too @Sriraj-dev can help in that department

**Comment by king-11** [comment]
For placement portal use the landing image from here https://www.placement.iitbhu.ac.in/

**Comment by king-11** [comment]
The telegram one is getting pixelated let's crop it

**Comment by eshaanagarwal** [comment]
@king-11 All mobile (vertical) screen shots are getting pixelated. Should i crop all of them ?

**Comment by king-11** [comment]
Yeah let's do that for discord bot you can take ss on command from chatroom on desktop/browser

**Comment by king-11** [comment]
Apart from telegram bot and study portal rest all images are good 🤔

**Review by king-11 — APPROVED** [review]

**Pull request merged**